tests/acceptance/projects-test.js

```diff
@@ -32,5 +32,8 @@ test('It renders all the required ui elements', (assert) => {
   andThen(function() {
     assert.equal(find('.project-list').length, 1, 'project-list component is rendered');
     assert.equal(find('.project-list .project-item').length, 5, 'correct number of project-items is rendered');
+
+    let firstProjectHref = find('.project-list .project-item:eq(0) a').attr('href');
+    assert.ok(firstProjectHref.indexOf('/test_organization/test_project_0') > -1, 'The link to a project is properly rendered');
```

Comment:
This is the assertion that's failing, since I didn't manage to fix the link-to issue here yet. If we end up going in a different direction, feel free to remove it.